src/js/ctrl/export.ts

```diff
@@ -49,9 +49,20 @@ export function exporter(internal: boolean = false): Spec {
   // Add interactions and widgets from store
   spec = exporter.interactions(state, spec);
   spec = exporter.widgets(state, spec);
+
+  if (state.getIn(['vis', 'present', 'facetLayouts']).size > 0){
+    spec.layout = exporter.layouts(state, int);
+  }
   return spec;
 }
 
+exporter.layouts = function (state: State, internal: boolean) {
+  const facetLayouts = state.getIn(['vis', 'present', 'facetLayouts']);
+  const layout = clean(duplicate(facetLayouts), internal);
+  const id = Object.keys(layout)[Object.keys(layout).length -1];
+  return layout[id];
+}
```

Comment:
if i'm understanding this correctly, the logic is:
- if there are any facet layouts in the lyra store, set the last one created as the layout in the exported spec

a few questions to help my understanding:
- why do we discard the previous ones?
- what are the cases where there will be more than one facet layout?
- how is this different from how non-facet layouts are handled?

Reply:
Really wasn't sure on how the logic of the export worked so this might be something we can walk through tomorrow